_bibliography/references.bib

```diff
@@ -1,2419 +1,4483 @@
+	author = {Huff, Kathryn D.},
+	month = feb,
+	year = {2019},
+}
+
+@misc{chaube_dynamic_2019,
+	address = {I2CNER Annual Symposium, Fukuoka, Japan},
+	title = {Dynamic {Transition} {Analysis} with {TIMES}},
+	url = {http://arfc.github.io/pres/2019-02-01-chaube-i2cner.pdf},
+	author = {Chaube, Anshuman},
+	month = feb,
+	year = {2019},
+}
+
+@inproceedings{bae_impact_2018-1,
+	address = {Orlando, FL},
+	title = {Impact of {Composition} {Approximation} on {Simulated} {Nuclear} {Fuel} {Cycle} {Metrics}},
+	url = {http://arfc.npre.illinois.edu/pres/2018-11-13-bae-answinter2018.pdf},
+	booktitle = {Proceedings of the {American} {Nuclear} {Society} {Winter} {Meeting}},
+	publisher = {American Nuclear Society},
+	author = {Bae, Jin Whan and Peterson-Droogh, Joshua L. and Huff, Kathryn D.},
+	month = nov,
+	year = {2018},
+}
+
+@article{bae_synergistic_2019,
+	title = {Synergistic spent nuclear fuel dynamics within the {European} {Union}},
+	volume = {114},
+	issn = {0149-1970},
+	url = {http://www.sciencedirect.com/science/article/pii/S014919701930037X},
+	doi = {10.1016/j.pnucene.2019.02.001},
+	abstract = {The French 2012–2015 Commission Nationale d’Evaluation Reports emphasize preparation for a transition from Light Water Reactors (LWRs) to Sodium-Cooled Fast Reactors (SFRs). We used the Cyclus nuclear fuel cycle simulator to explore the feasibility of enabling a French transition to an SFR fleet by using Used Nuclear Fuel (UNF) from other European Union (EU) nations. A Cyclus simulation captured nuclear power deployment in the EU from 1970 to 2160. In this simulation, France begins its planned transition to SFRs as existing LWRs are decommissioned. These SFRs are fueled with UNF accumulated by other EU nations and reprocessed in France. The impact of reactor lifetime extensions and SFR breeding ratios on time-to-transition were investigated with additional simulations. These simulations demonstrate that France can avoid deployment of additional LWRs by accepting UNF from other EU nations, that lifetime extensions delay time-to-transition, and improved breeding ratios are not particularly impactful.},
+	urldate = {2019-04-15},
+	journal = {Progress in Nuclear Energy},
+	author = {Bae, Jin Whan and Singer, Clifford E. and Huff, Kathryn D.},
+	month = jul,
+	y
```

Comment:
This one was a seminar at UTK, which I think should be the address, and not UIUC.